src/main/java/com/parasoft/demoapp/graphql/ItemGraphQLDataFetcher.java

```diff
@@ -82,4 +80,24 @@ public DataFetcher<PageInfo<ItemEntity>> getItems() {
             }
         };
     }
+
+    public DataFetcher<ItemEntity> getItemByItemId() {
+        return dataFetchingEnvironment -> {
+            try {
+                Map<String, String> uriVariables = new HashMap<>();
+                if (dataFetchingEnvironment.containsArgument("itemId")) {
+                    uriVariables.put("itemId", dataFetchingEnvironment.getArgument("itemId"));
+                }
```

Comment:
Please change this block to be:
<pre>String itemId= dataFetchingEnvironment.getArgument("itemId");
if (itemId!= null && !itemId.isEmpty()) {
    uriVariables.put("itemName", itemId);
}</pre>